datasets/indonlu/README.md

```diff
@@ -0,0 +1,594 @@
+|    | dataset | annotations_creators | language_creators | languages | licenses | multilinguality | size_categories | source_datasets | task_categories      | task_ids                                                  |
```

Comment:
The tags must be in YAML, not markdown

Can you fix that please ?

You can use the tagger app to generate the yaml. More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Reply:
Thanks for the review @lhoestq.
I have already made some changes to your review. But, I ended up creating a new PR in here #1530, kindly check for reviews, thank you 🤗